utils/version-check.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ver=$1
+
+red='\e[0;41m'
+RED='\e[1;31m'
+green='\e[0;32m'
+GREEN='\e[1;32m'
+NC='\e[0m'
+
+
+# doc: apisix $ver
+
+matched=`grep "apisix-dashboard-v[0-9][0-9.]*" -r docs/`
+expected=`grep "apisix-dashboard-v$ver" -r docs/`
```

Comment:
What's the difference between `matched` and `expected`?

Reply:
List the content that matches `apisix-dashboard-v[0-9][0-9.]*` to see if it is the same as the specified version. If it is not the same, we can assume that the version number of such content has not been modified.